### ISSUE TITLE
how to make the pretrained model using coarse annotation?

### ISSUE THREAD

**Issue by youngwanLEE** [MSiam/TFSegmentation#6]
Hi, I was really impressed by your works.

I wonder how to make the pretrained model using coarse annotation as described in your paper.

thanks in advance.

**Comment by MSiam** [comment]
1- You need to have the training data with coarse annotations they are 20,000 images. Example: Construct directory cityscapes with this structure:
cityscapes:
      ----> images--->train
      ---->labels--->train

2- Modify run.sh to run the coarse training.
Uncomment line 24 for training SkipNet-MobileNet on coarse data.

3- I have already pushed the config file we use for coarse training.
Make sure you modify the abs_data_dir:  /path_to_cityscapes/cityscapes/ on your machine

4- Note the main change in the config file is the use of data_mode "experiment_tfdata". Since the data is larger than the fine we cant load them all in memory so tf.data is used for efficient loading.

Let us know if further issues occur. We will soon upload more details to the readme and config files for running the rest of the models.
